src/frontend/go.mod

```diff
@@ -5,34 +5,24 @@ go 1.19
 require (
 	cloud.google.com/go/compute v1.10.0
 	cloud.google.com/go/profiler v0.3.0
-	contrib.go.opencensus.io/exporter/jaeger v0.2.1
-	contrib.go.opencensus.io/exporter/stackdriver v0.13.12
 	github.com/golang/protobuf v1.5.2
 	github.com/google/uuid v1.3.0
 	github.com/gorilla/mux v1.8.0
 	github.com/pkg/errors v0.9.1
 	github.com/sirupsen/logrus v1.9.0
-	go.opencensus.io v0.23.0
 	golang.org/x/net v0.0.0-20220927171203-f486391704dc
 	google.golang.org/grpc v1.49.0
 )
 
 require (
 	cloud.google.com/go v0.104.0 // indirect
-	cloud.google.com/go/monitoring v1.1.0 // indirect
-	cloud.google.com/go/trace v1.0.0 // indirect
-	github.com/aws/aws-sdk-go v1.43.31 // indirect
-	github.com/census-instrumentation/opencensus-proto v0.3.0 // indirect
 	github.com/golang/groupcache v0.0.0-20210331224755-41bb18bfe9da // indirect
 	github.com/google/go-cmp v0.5.8 // indirect
 	github.com/google/pprof v0.0.0-20220412212628-83db2b799d1f // indirect
 	github.com/googleapis/enterprise-certificate-proxy v0.1.0 // indirect
 	github.com/googleapis/gax-go/v2 v2.4.0 // indirect
-	github.com/jmespath/go-jmespath v0.4.0 // indirect
-	github.com/prometheus/prometheus v2.5.0+incompatible // indirect
-	github.com/uber/jaeger-client-go v2.25.0+incompatible // indirect
+	go.opencensus.io v0.23.0 // indirect
```

Comment:
Where does this one come from?

Reply:
According to ` go mod why -m go.opencensus.io` this comes from this chain:
```
github.com/GoogleCloudPlatform/microservices-demo/src/frontend
cloud.google.com/go/profiler
google.golang.org/api/transport/grpc
go.opencensus.io/plugin/ocgrpc
```
Though it's possible there are other dependency paths.  I imagine we'll address that in a future PR.